src/sidebar/templates/annotation-header.html

```diff
@@ -1,26 +1,14 @@
 <header class="annotation-header">
   <!-- User -->
   <span ng-if="vm.user()">
-    <a class="annotation-header__user"
-      target="_blank"
-      ng-if="!vm.isThirdPartyUser()"
-      ng-href="{{vm.serviceUrl('user',{user:vm.user()})}}"
-      >{{vm.displayName()}}</a>
-    <a class="annotation-header__user"
-      target="_blank"
-      ng-if="vm.isThirdPartyUser() && vm.thirdPartyUsernameLink()"
-      href="{{ vm.thirdPartyUsernameLink() }}"
-      >{{vm.displayName()}}</a>
-    <span class="annotation-header__user"
-      ng-if="vm.isThirdPartyUser() && !vm.thirdPartyUsernameLink()"
-      >{{vm.displayName()}}</span>
+    <annotation-user annotation="vm.annotation"></annotation-user>
+    <br />
```

Comment:
This `<br />` used to occur _after_ the `annotation-collapsed-replies` class. It now appears _before_ it. You can see the effect of this if you:

1. Create an annotation
2. Create a reply to that annotation
3. Create a reply to the reply
4. Collapse the reply created in step (2), and look at the "1 reply" message which should appear next to the username rather than under it:

<img width="454" alt="collapsed-replies-label" src="https://user-images.githubusercontent.com/2458/60164575-c1759100-97f5-11e9-9048-6d716476711f.png">